runtime/src/committee.rs

```diff
@@ -0,0 +1,752 @@
+        pub const TransactionBaseFee: u64 = 0;
+        pub const TransactionByteFee: u64 = 0;
+    }
+
+    impl balances::Trait for Test {
+        type Balance = u128;
+        type OnFreeBalanceZero = ();
+        type OnNewAccount = ();
+        type Event = ();
+        type TransactionPayment = ();
+        type DustRemoval = ();
+        type TransferPayment = ();
+        type ExistentialDeposit = ExistentialDeposit;
+        type TransferFee = TransferFee;
+        type CreationFee = CreationFee;
+        type TransactionBaseFee = TransactionBaseFee;
+        type TransactionByteFee = TransactionByteFee;
+        type WeightToFee = ConvertInto;
+        type Identity = identity::Module<Test>;
+    }
+
+    parameter_types! {
+        pub const MinimumPeriod: u64 = 3;
+    }
+
+    impl timestamp::Trait for Test {
+        type Moment = u64;
+        type OnTimestampSet = ();
+        type MinimumPeriod = MinimumPeriod;
+    }
+    #[derive(codec::Encode, codec::Decode, Debug, Clone, Eq, PartialEq)]
+    pub struct TestProposal {
+        pub dummy: u8,
+    }
+
+    impl sr_primitives::traits::Dispatchable for TestProposal {
+        type Origin = Origin;
+        type Trait = Test;
+        type Error = DispatchError;
+
+        fn dispatch(self, _origin: Self::Origin) -> DispatchResult<Self::Error> {
+            Ok(())
+        }
+    }
+
+    impl identity::Trait for Test {
+        type Event = ();
+        type Proposal = TestProposal;
+        type AcceptTickerTransferTarget = Test;
+    }
+
+    impl crate::asset::AcceptTickerTransfer for Test {
+        fn accept_ticker_transfer(_: IdentityId, _: u64) -> srml_support::dispatch::Result {
+            unimplemented!()
+        }
+    }
+
+    type Identity = identity::Module<Test>;
+    type AccountId = <AnySignature as Verify>::Signer;
+
+    parameter_types! {
+        pub const CommitteeOrigin: AccountId = AccountId::from(AccountKeyring::Alice);
+    }
+
+    impl Trait<Instance1> for Test {
+        type Origin = Origin;
+        type Proposal = Call;
+        type CommitteeOrigin = EnsureSignedBy<CommitteeOrigin, AccountId>;
+        type Event = ();
+    }
+
+    impl Trait for Test {
+        type Origin = Origin;
+        type Proposal = Call;
+        type CommitteeOrigin = EnsureSignedBy<CommitteeOrigin, AccountId>;
+        type Event = ();
+    }
+
+    pub type Block = sr_primitives::generic::Block<Header, UncheckedExtrinsic>;
+    pub type UncheckedExtrinsic = sr_primitives::generic::UncheckedExtrinsic<u32, u64, Call, ()>;
+
+    srml_support::construct_runtime!(
+		pub enum Test where
+			Block = Block,
+			NodeBlock = Block,
+			UncheckedExtrinsic = UncheckedExtrinsic
+		{
+			System: system::{Module, Call, Event},
+			Committee: committee::<Instance1>::{Module, Call, Event<T>, Origin<T>, Config<T>},
+			DefaultCommittee: committee::{Module, Call, Event<T>, Origin<T>, Config<T>},
+		}
+	);
+
+    fn make_ext() -> sr_io::TestExternalities<Blake2Hasher> {
```

Comment:
If you use `TestStorage` you could keep this as local or update `build_ext()` in `runtime/src/test/storage.rs`

Reply:
Same reasons here too. Please see above.